scripts/cypress/integration/common/3b_formula_column.js

```diff
@@ -220,6 +229,16 @@ export const genTest = (apiType, dbType) => {
         Math.sqrt(countryId[i])
       );
 
+      RESULT_DATE_DIFF_0[i] = 1440;
+      RESULT_DATE_DIFF_1[i] = 86400;
+      RESULT_DATE_DIFF_2[i] = 86400000;
+      RESULT_DATE_DIFF_3[i] = 24;
+      RESULT_DATE_DIFF_4[i] = 52;
+      RESULT_DATE_DIFF_5[i] = 12;
+      RESULT_DATE_DIFF_6[i] = 4;
+      RESULT_DATE_DIFF_7[i] = 1;
+      RESULT_DATE_DIFF_8[i] = 365;
```

Comment:
none of them is correct.

Here's the expected values from Airtable using the formula `DATETIME_DIFF("2022/10/14", "2022/10/15")` with different units. As I mentioned in previous comments, given `date1` and `date2`, you should return `date1` - `date2` instead of `date2` - `date1`.

![image](https://user-images.githubusercontent.com/35857179/195972635-1ad6561e-4dc4-42da-9426-387a120f2be6.png)